src/core/linepolygonhighlight.h

```diff
@@ -60,12 +60,14 @@ class LinePolygonHighlight : public QQuickItem
     void updated();
 
   private slots:
+    void rotationChanged();
     void mapCrsChanged();
     void visibleExtentChanged();
     void makeDirty();
 
   private:
     virtual QSGNode *updatePaintNode( QSGNode *n, UpdatePaintNodeData * ) override;
+    virtual void updateTransform();
```

Comment:
Why is that virtual, I don't see any overrides?

Reply:
@m-kuhn , no you are right, this was a hasty copy/paste typo (from the line above).